File: code/algo/getoffer/c++/2/check_nosort.cpp
#include <string>
#include <vector>

bool isAnagram(const std::string &s, const std::string &t) {
  if (s == t) {
    return false;
  }
  std::vector<int> c1(26, 0), c2(26, 0);
  for (char ch : s) {
    c1[ch - 'a']++;
  }
  for (char ch : t) {
    c2[ch - 'a']++;
  }
  return c1 == c2;
}

File: code/algo/getoffer/go/2/check_nosort.go
package main

func isAnagram(s, t string) bool {
	if s == t {
		return false
	}
	var c1, c2 [26]int
	for _, ch := range s {
		c1[ch-'a']++
	}
	for _, ch := range t {
		c2[ch-'a']++
	}
	return c1 == c2
}
